ramalama/huggingface.py

```diff
@@ -33,9 +33,8 @@ def fetch_checksum_from_api(url):
 
 class Huggingface(Model):
     def __init__(self, model):
-        model = model.removeprefix("huggingface://")
-        model = model.removeprefix("hf://")
-        model = model.removeprefix("hf.co/")
+        model = rm_until_substring(model, "hf.co/")
```

Comment:
Why are you dropping hugginface://?

Reply:
It's not dropped, at this point of the code, we have already verified the start of the protocol is fine, so we do a more generic:

```
model = rm_until_substring(model, "://")
```

Reply:
```
if model.startswith("huggingface://") or model.startswith("hf://") or model.startswith("hf.co/"):
```